chess-AI/output_representation.py

```diff
@@ -91,14 +93,14 @@ def convert_policy_indices_to_uci_move(self, indices, board_t):
             n_squares = -n_squares
 
         move_directions = {
-            "N": np.array([0, n_squares]),
-            "S": np.array([0, -n_squares]),
-            "E": np.array([n_squares, 0]),
-            "W": np.array([-n_squares, 0]),
-            "NE":np.array([n_squares, n_squares]),
-            "NW":np.array([-n_squares, n_squares]),
-            "SW":np.array([-n_squares, -n_squares]),
-            "SE":np.array([n_squares, -n_squares]),
+            "N":  np.array([0, n_squares]),
+            "S":  np.array([0, -n_squares]),
+            "E":  np.array([n_squares,  0]),
+            "W":  np.array([-n_squares, 0]),
+            "NE": np.array([n_squares,   n_squares]),
+            "NW": np.array([-n_squares,  n_squares]),
+            "SW": np.array([-n_squares, -n_squares]),
+            "SE": np.array([n_squares,  -n_squares]),
```

Comment:
Can you pls make these all just have one space between colon and text and between commas and stuff?